pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -196,7 +196,11 @@ func getKubeadmConfig(cfg *config.Cluster, data kubeadm.ConfigData, node nodes.N
 			// order matters since the nodeAddress will be used later to configure the apiserver advertise address
 			// Ref: #2484
 			primaryServiceSubnet := strings.Split(cfg.Networking.ServiceSubnet, ",")[0]
```

Comment:
we should probably split this out into a helper w/ tests at some point

Reply:
yeah, I started to do that, and ended moving too much code for something I'd like to get working fast XD